benchmarks/dynamo/common.py

```diff
@@ -4377,9 +4377,6 @@ def run(runner, args, original_dir=None):
             # Set translation validation on by default on CI accuracy runs.
             torch.fx.experimental._config.translation_validation = True
 
-        ci = functools.partial(
```

Comment:
Is this a bug? cc @xuzhao9

Reply:
I think it's just cruft.

`CI` is just a dataclass, so this is just a shorthand to creating an instance of `CI` - so it's not like some task is being forgotten to be done.